client/src/pages/Newspage.js

```diff
@@ -0,0 +1,73 @@
+import styled from 'styled-components';
+import Footer from '../components/Footer';
+
+export default function Newspage({ news }) {
+  return (
```

Comment:
I think the `return` is well written and good to understand right away. Good job! If you want to shorten your js file, you could think about putting `news.map etc.` into a new `NewspageNews.js` but that's up to you.